DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/ViewDelegate/ConfirmRemoveFromCourse.cshtml

```diff
@@ -14,25 +14,41 @@
     @if (errorHasOccurred) {
       <vc:error-summary order-of-property-names="@(new[] { nameof(Model.Confirm) })" />
     }
-    <h1 class="nhsuk-heading-xl nhsuk-u-margin-bottom-8 word-break">Remove enrolment</h1>
-    <h2 class="nhsuk-heading-l word-break">Delegate: @Model.Name</h2>
-    <h2 class="nhsuk-heading-l word-break">Activity: @Model.CourseName</h2>
-    <div class="nhsuk-form-group">
-      <form asp-action="ExecuteRemoveFromCourse" asp-route-delegateId="@Model.DelegateId" asp-route-customisationId="@Model.CustomisationId">
-        <fieldset class="nhsuk-fieldset" aria-describedby="example-hint">
-          <div class="nhsuk-checkboxes">
-            <div class="nhsuk-checkboxes__item">
-              <input type="hidden" name="name" value="@Model.Name" />
-              <input type="hidden" name="courseName" value="@Model.CourseName" />
-              <vc:single-checkbox asp-for="@nameof(Model.Confirm)"
-                                  label="I am sure that I wish to remove this enrolment"
-                                  hint-text="This action will remove the activity from the delegate’s Current Activities list and archive their progress." />
-            </div>
-          </div>
-          <button class="nhsuk-button delete-button nhsuk-u-margin-top-4 nhsuk-u-margin-bottom-6" role="button" type="submit">Remove enrolment</button>
-        </fieldset>
-      </form>
-      <vc:cancel-link asp-controller="ViewDelegate" asp-action="Index" asp-all-route-data="@cancelLinkRouteData" />
-    </div>
+    <h1 class="nhsuk-heading-xl">Remove enrolment</h1>
+  </div>
+</div>
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-one-quarter nhsuk-heading-l nhsuk-u-font-weight-bold">
+    Delegate:
+  </div>
+  <div class="nhsuk-grid-column-three-quarters nhsuk-heading-l nhsuk-u-font-weight-normal word-break">
+    @Model.Name
+  </div>
+</div>
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-one-quarter nhsuk-heading-l nhsuk-u-font-weight-bold">
+    Activity:
+  </div>
+  <div class="nhsuk-grid-column-three-quarters nhsuk-heading-l nhsuk-u-font-weight-normal word-break">
+    @Model.CourseName
+  </div>
+</div>

```

Comment:
We should have a partial in "Views/TrackingSystem/Delegates/Shared/_DelegateAndCourseNames.cshtml" that might help reduce the code here a bit, though you might want to double check that the styling is exactly what we want. If it isn't we might need to see where else the styling isn't consistent as that partial was only recently introduced.

Reply:
The styling is slightly different (the top one is using the partial)
Do we want to the make partial consistent with the styling used on the remove enrolment page? The partial is currently only used in the progress/learning log.

![image](https://user-images.githubusercontent.com/58184056/157238811-aed339d3-8149-4b7c-9591-de548e4dc7a8.png)
![image](https://user-images.githubusercontent.com/58184056/157238922-cde01ecc-5cb4-4833-8467-e55b833822a4.png)

Reply:
I'd say because this is relating to a delegate-course relation we use the partial here as it is. 
There are a number of other places where we display the delegate name on its own in a similar style, but it looks like the font weight varies between these.

At this point I think we should just use the partial as it is, and not worry about consistency here, but update this ticket to mention the issue there: https://softwiretech.atlassian.net/browse/HEEDLS-785

@AlexJacksonDS as you were the one that spotted the issue when testing, do you have much of a preference as to the boldness here?

Reply:
Are we not ditching this layout in https://github.com/TechnologyEnhancedLearning/DLSV2/pull/989 in favour of the card on the progress pages?

I think the bold is better on the field name in general

Reply:
Good point. In which case, use bold on the field name, with normal-weighted values. I'll make a note for myself to have a look through for places that aren't in the style and add them to that ticket I mentioned.